src/main/java/ca/corefacility/bioinformatics/irida/model/MutableIridaThing.java

```diff
@@ -3,34 +3,33 @@
 import java.util.Date;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
 
 /**
  * An object that can be modified
  */
 public interface MutableIridaThing extends IridaThing {
-	
+
 	/**
 	 * Set the numerical identifier for this object
-	 * 
-	 * @param id
-	 *            The ID to set
+	 *
+	 * @param id The ID to set
 	 */
 	@JsonProperty("identifier")
 	public void setId(Long id);
-	
 
 	/**
 	 * Get the date that this object was last modified
-	 * 
+	 *
 	 * @return {@link Date} object of the modified date
 	 */
+	@Schema(implementation = Long.class)
```

Comment:
Could you add a description here indicating that it is Epoch time e.g.:
```Epoch time in milliseconds when the resource was modified```

Reply:
Please see a922b821d63d07ad1000087b7880ef91f815c1a4.